homeassistant/components/qnap_qsw/strings.json

```diff
@@ -52,7 +52,16 @@
       },
       "uptime": {
         "name": "Uptime"
+      },
+      "uptime_timestamp": {
+        "name": "Uptime timestamp"
       }
```

Comment:
Hmm, something in me things uptime is viable for both, but I'm not sure what is best here

Reply:
I'm happy with both, so I can change it to use uptime for both if you prefer that.

Reply:
The only downside is that they'll both be called uptime so one will be appended `_2`

Reply:
> The only downside is that they'll both be called uptime so one will be appended `_2`

Ah, you're right, and we won't be able to tell them appart for `uptime_seconds_deprecated`...

Reply:
I've asked for a second opinion on this. 

But also keep in mind, people could've renamed both the name and entity id, so it could but even be recognizable